3_3sum.java

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
Hey Cindy,
               I am guessing you are missing out on import statements. Also I might be wrong but i was unable to find the main method, i was wondering if I am missing something here.